docs/source/conf.py

```diff
@@ -18,6 +18,13 @@
 import subprocess
 import sys
 
+try:
+    from queue import Queue, Empty
+except ImportError:
+    from Queue import Queue, Empty
```

Comment:
we use `six.moves` for that:
```suggestion
from six.moves.queue import Queue, Empty
```

But it seems unused, so you can safely skip changing this file altogether.

Reply:
cosmetic tips, but still:

The following code:

```python
from FooBar import something
```

Has a giant problem `FooBar`'s CamelCaps means the associated file is called `FooBar.py` that in turn is verrrrrrrrrrry rare.

In other words, the line:

```python
from Queue import Queue, Empty
```

Is verrrrrrrrrrrrrrrrry unlikely to be correct.

Python filename convention is `snake_case.py`, hence module name is `snake_case` too.

Reply:
I trough it was python2 VS 3. I am going to test.